src/augeas.c

```diff
@@ -83,7 +83,8 @@ static const char *const errcodes[] = {
     "Failed to execute command",                        /* AUG_ECMDRUN */
     "Invalid argument in function call",                /* AUG_EBADARG */
     "Invalid label",                                    /* AUG_ELABEL */
-    "Cannot copy node into its descendant"              /* AUG_ECPDESC */
+    "Cannot copy node into its descendant",             /* AUG_ECPDESC */
+    "Cannot setup file filter"                          /* AUG_ESFILTER */
```

Comment:
The way this is used, wouldn't `Cannot find a lens for the given file` be more accurate ?